src/components/Header.astro

```diff
@@ -0,0 +1,87 @@
+---
+import { Image } from "astro:assets"
+import Logo from "../assets/logo-round.svg"
+import ExternalLink from "../assets/icons/external-link.svg"
+---
+
+<header class="link-card">
+	<nav class="nav">
+		<div class="logo-wrapper">
+		<a href="https://nytimesguild.org/" class="logo">
+			<Image class="logo-img"src={Logo} format="svg" alt="" loading="lazy" />
+			<span class="sr-only">Times Tech Guild</span>
+		</a>
+		<h1 class="logo-text">Guild Builds</h1>
+	</div>
+		<a class="donate-link" href="https://www.gofundme.com/f/nyt-tech-strike-fund" target="_blank" rel="noopener">
+			Donate
+			<Image class="donate-link-img" src={ExternalLink} format="svg" alt="" height="16" />
+			<span class="sr-only"> opens a new window</span>
+		</a>
+	</nav>
+	
+</header>
+<style>
+	.nav {
+		padding: 1rem;
+		display: flex;
+		align-items: center;
+		background: #290E0D;
+		flex-direction: column;
+		border-bottom: 1px solid #B42F2F;
+	}
+	.logo-wrapper {
+		display: flex;
+	}
+	.logo {
+		display: flex;
+	}
+	.logo-img {
+		height: 2.625rem;
+		width: 2.625rem;
+	}
+	.logo-text {
+		font-family: "Crimson Pro", serif;
+		font-size: 2rem;
+		font-style: italic;
+		font-weight: 800;
+		color: #FFFFFF;
+		margin-left: 16px;
+	}
+	.donate-link {
+		display: flex;
+		align-items: center;
+		color: #FFFFFF;
+		font-family: "Open Sans", sans-serif;
+		font-weight: 600;
+		font-size: 1.25rem;
+		text-decoration: none;
+		border: 2px solid #FF4040;
+		padding: 0.375rem 1rem;
+	}
+	.donate-link:hover {
+		text-decoration: underline;
+	}
+	.donate-link-img {
+		margin-left: 16px;
+		height: 1.625rem;
+		width: 1.625rem;
+	}
+	.sr-only {
```

Comment:
Nitpick: this could should probably end up in global styles, as I'm sure we'll use it somewhere else.